### ISSUE TITLE
For discussion: Update video-miners section in preparation for the merge

### ISSUE THREAD

**Issue by chrishobcroft** [livepeer/docs#185]
Currently the /video-miners section of the docs is pitched in terms of helping GPU operators "increase profits by transcoding video on GPUs _while mining_". This implies that you need to be mining, and might evem put off operators who don't wish to engage in PoW, as it isn't immediately clear that you can transcoded without also mining.

AFAIK there are not so many Os who also mine, plus the economics may even work without hashing at the same time as transcoding. Thus, it feels like this section needs a refresh.

Factor in the upcoming switch-off of PoW on Ethereum, GPU operators are going to be looking for ways to re-purpose their hardware.

Clearly, Ethereum isn't the only network that needs hashing, but there is perhaps an opportunity to expand on the green credentials, e.g. "using GPUs for their intended purpose i.e. processing graphics (what a radical concept /s)" instead of "proof-of-waste".

Discuss?

**Comment by hthillman** [comment]
To be frank I'm not particularly knowledgeable about the energy consumption of GPUs on the Livepeer Network - maybe you know more? If we're going to try saying that we're better for the environment than PoW, we need to have the numbers to back it up. I'm not sure "hey, at least we're cranking out video instead of blocks" is a compelling environmental argument.

That said, I *do* think this section needs updates before the merge, and I'm not convinced that "video miners" is even the right terminology to use.

Curious for thoughts from @papabear99

**Comment by chrishobcroft** [comment]
Yeah, my climate argument was entirely about "at least we're doing something directly useful when burning the energy" i.e. to help communication reach more people. There's got to be a positive angle re: aiding communication in this closed-minded world.

But compared to PoW hashing, (not "cranking out blocks" - that bit doesn't require much energy) which is of zero benefit in light of PoS, then transcoding does have some non-zero benefit. Whether this non-zero benefit is worthwhile is another question.

Interested to know is @0xVires has a take on this.

Oh, and I have always thought that "video miner" is a terrible way to brand it. It immediately puts you on the back-foot, having to explain that "it's _not_ like PoW mining".

**Comment by papabear99** [comment]
I did some initial testing regarding power consumption when I first started up to make sure my solar array and batteries could handle the load.  The setup I was running with 2 3070 GPUs with an i9 9900 was drawing 110-120 watts from the wall while transcoding vs ~300 mining ETH and didn't require external cooling.  There is also a big energy savings when you consider that POW is running at max consumption 24/7 whereas an Orchestrator/Transcoder throttles down the power when demand drops with cards like a Quadro RTX 4000 only consuming ~6 watts when idle.

My take on pitching transcoding as something that can be done while mining is, that while technically true in practice provides less than optimal results and shouldn't be a promoted method for running transcoders on Livepeer.  RE @chrishobcroft's comment "...the economics may even work without hashing at the same time as transcoding" the economics are much better using a GPU for transcoding (assuming it has work) than mining ETH.
